### PR TITLE
Add more craft icons: carpenter, sawmill, brewery

### PR DESCRIPTION
**Pull request by dvdmrtnz**
This PR adds more icons for https://github.com/organicmaps/organicmaps/issues/2748. These icons have been made by myself.

Carpenter:
 

Sawmill:

### CODE REVIEW

File: data/styles/clear/include/Icons.mapcss [hunk 1]
```diff
@@ -2619,6 +2622,9 @@ area|z17-[craft=photographer]
 node|z17-[craft=plumber],
 area|z17-[craft=plumber]
 {icon-image:plumber-m.svg;}
+node|z17-[craft=sawmill],
+area|z17-[craft=sawmill]
+{icon-image:sawmill-m.svg;}
```
Comment (vng) [inline data/styles/clear/include/Icons.mapcss:2627]: This changes should be in first commit with icons, not in "Regenerated".

**Review by vng — CHANGES_REQUESTED** [review]

File: data/styles/clear/include/Icons.mapcss (hunk 1 above)
Comment (dvdmrtnz) [reply data/styles/clear/include/Icons.mapcss:2627]: You're right. Fixed!

**Review by vng — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
@vng will users see these icons in the upcoming data update? Or maps should be regenerated again?

**Comment by dvdmrtnz** [comment]
I added the brewery icon from https://github.com/organicmaps/organicmaps/issues/2187 to this PR too.

**Comment by vng** [comment]
@biodranik Yes, they will. Do you like icons?

**Pull request merged**